lib/vanagon/component.rb

```diff
@@ -290,6 +290,14 @@ def get_build_dir
       end
     end
 
+    def get_dependency_hash
+      obj = @options
+      %i[version name].each do |var|
+        obj[var.to_s] = self.instance_variable_get("@#{var}")
```

Comment:
I'm inclined to agree with Nate. No reason to even add iteration here:

```ruby
def get_dependency_hash
  { name => { options[:name], options[:version] } }
end
```

That's pretty gosh darned readable